docs/source/conf.py

```diff
@@ -0,0 +1,55 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'geoextent'
+copyright = '2019, o2r-project'
+author = 'o2r-project'
+
+# The full version, including alpha/beta/rc tags
+release = '0.0.1'
```

Comment:
You can get this information from the one place where it is configured, see https://github.com/jupyter/repo2docker/blob/master/docs/source/conf.py#L76 and following lines.

Reply:
`Dev-readme.md` now is part of docs and `make.bat` is removed.

Reply:
Good. Could you also fix the version to be imported from (from `setup.py`) ?

Reply:
- Import version from setup.py
   - [ ] getting to know how it is implemented in `repo2docker` [https://github.com/jupyter/repo2docker/blob/master/docs/source/conf.py#L76]
   - [ ] going through `python-versioneer` [https://github.com/warner/python-versioneer]
   - [ ] found an easy way to do it in [https://stackoverflow.com/questions/26141851/let-sphinx-use-version-from-setup-py]